templates/module/helfi_tpr/tpr-unit--teaser-with-image.html.twig

```diff
@@ -12,8 +12,15 @@
   {% endif %}
 {% endset %}
 
+{%
+  set card_tpr_classes = [
+    not unit_published ? 'card--unpublished',
```

Comment:
This variable could be more generic. Like `not content_entity_published ? 'card--unpublished',`

Reply:
Made it generic 🦖